### PR TITLE
feat: Standardize issue templates for discussions

### PR DESCRIPTION
**Pull request by kellertk**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -1,50 +1,48 @@
...
   - type: textarea
     id: current
     attributes:
-      label: Current behavior
+      label: Current Behavior
       description: |
-        Tell us what happens instead of the expected behavior.
-
-        Include full errors, uncaught exceptions, stack traces, and relevant logs.
-        To turn on SDK logging, follow instructions here: http://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/java-dg-logging.html
```
Comment (debora-ito) [inline .github/ISSUE_TEMPLATE/bug-report.yml:31]: Can we put back the link to the SDK logging instructions?

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 2]
```diff
@@ -10,41 +12,39 @@ body:
...
   - type: checkboxes
-    id: acknowledgment
+    id: ack
     attributes:
-      label: Acknowledge
+      label: Acknowledgements
       options:
         - label: I may be able to implement this feature request
           required: false
+        - label: This feature might incur a breaking change
```
Comment (debora-ito) [inline .github/ISSUE_TEMPLATE/feature-request.yml:46]: Why is this checkbox needed? We don't make breaking changes, usually.

**Review by debora-ito — CHANGES_REQUESTED** [review]
Looks good overall, made a couple of comments.

Can you confirm that this pull request can be released under the Apache 2 license?

File: .github/ISSUE_TEMPLATE/feature-request.yml (hunk 2 above)
Comment (kellertk) [reply .github/ISSUE_TEMPLATE/feature-request.yml:46]: As an employee of Amazon, Amazon owns any IP I generate for this, so yes :)

Comment (kellertk) [reply .github/ISSUE_TEMPLATE/feature-request.yml:46]: But a proposed feature request might

**Comment by kellertk** [comment]
> Can you confirm that this pull request can be released under the Apache 2 license?

As an employee of Amazon, any IP that I generate is owned by Amazon, so yes. :)

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (kellertk) [reply .github/ISSUE_TEMPLATE/bug-report.yml:31]: New commit pushed

**Review by debora-ito — APPROVED** [review]

**Pull request merged**